I love clang-tidy too:
```
Apr 25 19:28:17 /build/src/Processors/Executors/ExecutionThreadContext.cpp:67:12: error: catch handler catches by value; should catch by reference instead [cert-err09-cpp,cert-err61-cpp,misc-throw-by-value-catch-by-reference,-warnings-as-errors]
Apr 25 19:28:17    67 |     catch (Exception exception)
Apr 25 19:28:17       |            ^
Apr 25 19:28:17 20171 warnings generated.
```

--- a/src/Processors/Executors/ExecutionThreadContext.cpp
+++ b/src/Processors/Executors/ExecutionThreadContext.cpp
@@ -64,7 +64,7 @@ static void executeJob(ExecutingGraph::Node * node, ReadProgressCallback * read_
             }
         }
     }
-    catch (Exception exception)
+    catch (Exception exception) /// NOLINT
     {
         /// Copy exception before modifying it because multiple threads can rethrow the same exception
         if (checkCanAddAdditionalInfoToException(exception))

